Resolve 23 ISO comments (#636)
buildingSMART/IFC4.3.x-development: docs/templates/Object Association/Material Association/Material Profile Set Usage/README.md

Material Profile Set Usage
==========================

> A material profile set usage may be defined on an object type. In this scenario, all occurrences of that type must use this profile set to parametrically define their geometry.

Reviewer: ```suggestion
When the occurrence has an associated IfcObjectType, that object type may define the material profile set (no usage). In this scenario, all occurrences of that type must use that profile set.
```

And move to bottom incl. note.

Include introductory sentence:

The material profile can be used to model an element as a set of prismatic element (for example, beam or column) by known cross section profiles.
